runtime/near-vm-runner/src/tests/runtime_errors.rs

```diff
@@ -418,7 +418,8 @@ fn test_stack_instrumentation_protocol_upgrade() {
         )
         .method("f1")
         .protocol_features(&[
-            ProtocolFeature::CorrectStackLimit,
+            #[allow(deprecated)]
+            ProtocolFeature::_DeprecatedCorrectStackLimit,
```

Comment:
(aside) @nagisa, could you please confirm if it's fine to remove checks for protocol features here? This file is similar to runtime/near-vm-runner/src/tests/compile_errors.rs

Reply:
(aside) Sure, but also why remove now if these tests don't start failing after the removal of the protocol version (due to them using parameter file configs to pick between implementations?)

If anything this actually makes it harder to spot and remove code that has been actually slated for removal, doesn't it? If we're removing a test like this, I would also expect the actual code implementing the old logic to be removed too (even if it is not logically directly gated by `ProtocolFeature::Foo.enabled()`)

Reply:
(aside) Alright, let me then revert this test. We can consider removing them later.